### PR TITLE
New package: Microsoft.PCManager version 3.9.4.0

### PR DESCRIPTION
**Pull request by schembriaiden**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154051)

### CODE REVIEW

File: manifests/m/Microsoft/PCManager/Beta/3.9.4.0/Microsoft.PCManager.Beta.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+ReleaseDate: 2023-12-04
+Installers:
+- Architecture: x64
+  InstallerUrl: https://files03.tchspt.com/down/MSPCManagerSetup.exe
```
Comment (stephengillie) [inline manifests/m/Microsoft/PCManager/Beta/3.9.4.0/Microsoft.PCManager.Beta.installer.yaml:17]: This doesn't appear to match the publisher, and this application appaers to not be released yet. Can you help me to find a link to this package on the publisher's site?

Comment (vikingnope) [reply manifests/m/Microsoft/PCManager/Beta/3.9.4.0/Microsoft.PCManager.Beta.installer.yaml:17]: https://www.techspot.com/downloads/7506-microsoft-pc-manager.html

Comment (stephengillie) [reply manifests/m/Microsoft/PCManager/Beta/3.9.4.0/Microsoft.PCManager.Beta.installer.yaml]: Exactly - that doesn't appear to be a Microsoft-owned domain. 
> © 1998 - 2024 TechSpot, Inc. All Rights Reserved.

https://www.techspot.com/about/
> TechSpot is a registered trademark and the website techspot.com is property of TechSpot, Inc.

This appears to be the domain of TechSpot, a different company.

**Pull request closed without merging**